Versioner/ObjC/ObjectiveC Extensions/ObjectiveCBaseExtensions.swift

```diff
@@ -26,6 +26,18 @@ public class UpdateResponse: NSObject {
     }
 }
 
+@objcMembers
+public class AppStoreResponse: NSObject {
```

Comment:
Why do we need `AppStoreResponse `?
It seems like it just holds the result without any other added value?

Reply:
Removed